sympy/utilities/lambdify.py

```diff
@@ -178,6 +178,102 @@ def _import(module, reload=False):
 # linecache.
 _lambdify_generated_counter = 1
 
+def pre_treatment_cse(args_f, expr):
```

Comment:
These functions should probably be private: append an underscore to the name: `_pre_treatment_cse()`.